editor/src/components/canvas/canvas-strategies/strategies/grid-draw-to-insert-strategy.tsx

```diff
@@ -218,9 +221,25 @@ const gridDrawToInsertStrategyInner =
             ? []
             : getWrappingCommands(insertedElementPath, maybeWrapperWithUid)
 
+        const isClick = interactionData.type === 'DRAG' && interactionData.drag == null
+
+        // for click-to-insert, don't use absolute positioning
+        const clickToInsertCommands = isClick
```

Comment:
the name clickToInsertCommands is misleading, these are just the commands to remove absolute positioning